packages/ffe-account-selector-react/src/components/base-selector/BaseSelector.spec.js

```diff
@@ -260,7 +261,7 @@ describe('<BaseSelector> methods', () => {
             .find('.ffe-base-selector__input-field')
             .simulate('change', event);
 
-        expect(onInputChange).toHaveBeenCalledWith(value);
+        expect(onInputChange).toHaveBeenCalledTimes(1);
```

Comment:
Funker det å fortsatt ha en `toHaveBeenCalledWith(value, event)` her? Gjerne i tillegg til `toHaveBeenCalledTimes(`

Reply:
Den får en sånn svær event-greie som jeg ikke vet hvordan jeg skal få mocka. Så at de andre stedene hvor det er simulert change med event at de hadde gjort det sånn som jeg endra det til nå her. Men det var litt feigt av meg hehe. Vet du hvordan man kan mocke et helt input-event på riktig måte? Kan kanskje ta en ringings i morgen og se på det?

Reply:
Vi kan ta en ringings i morgen! :ok_hand:

Reply:
Kanskje jests `expect.objectContaining()` kan være en mulighet å teste på event-greia. Se https://jestjs.io/docs/expect#expectobjectcontainingobject

Reply:
Prøvde det uten hell. :cry:  Men kan jo være jeg brukte det på feil måte, kan se litt mer på det.

Reply:
Kanke noe slikt?
https://jestjs.io/docs/mock-functions#mock-property
```
expect(someMockFunction.mock.calls[0][1]).toBeInstanceOf(Event);
```

Reply:
Dette funker, men da får vi ikke testa simulering på samme måte som det er gjort i testen fra før av da. :/ 
`
        const value = 'test';
        const event = { target: { value } };
        const onChangeSpy = jest.fn();
        const component = mountBaseSelector({
            onChange: onChangeSpy,
        }).instance();

        component.onInputChange(event);
        expect(onChangeSpy).toHaveBeenCalledWith(value, event);
`

Reply:
Nu er ju allrede testerna der i enzyme men jag tror react-testing-library ville vart en fordel. Når vi tester onInputChange så synes jag det er en implmentasjonsdetalj. Som og teste en private metod i java. 
Det er god stund siden jag brukte enzyme så har dessverre ikke noen goda inspill annat en att jag gjerne hade sett det borta, men det er ikke mitt bslut og ta :)

https://www.thoughtworks.com/radar/languages-and-frameworks/enzyme

https://testing-library.com/docs/react-testing-library/intro/